Userland/Applications/FileManager/PropertiesWindowHashesTab.cpp

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2021, FrHun <frhun@t-online.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "PropertiesWindowHashesTab.h"
+#include <AK/Format.h>
+#include <Applications/FileManager/PropertiesWindowHashesHashWidgetGML.h>
+#include <Applications/FileManager/PropertiesWindowHashesTabGML.h>
+#include <LibCore/IODevice.h>
+#include <LibGUI/Clipboard.h>
+#include <LibThreading/BackgroundAction.h>
+#include <unistd.h>
+
+HashTab::HashTab(String const& path, bool const calculate_on_show)
```

Comment:
`calculate_on_show` is passed by value, so `const` is unnecessary:
```suggestion
HashTab::HashTab(String const& path, bool calculate_on_show)
```